lib/video_info/providers/vkontakte.rb

```diff
@@ -52,11 +58,15 @@ def duration
       private
 
       def _set_data_from_api
-        uri = open(url, options)
+        url = URI.parse("https://vk.com/al_video.php")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.